docs/docs-content/profiles/cluster-profiles/create-cluster-profiles/define-profile-variables.md

```diff
@@ -6,25 +6,25 @@ sidebar_position: 40
 tags: ["profiles", "cluster profiles"]
 ---
 
-Cluster profile variables enable you to create placeholders for parameters in profile layer configurations, which you
-can then populate for individual clusters during deployment. Meaning you can use a single cluster profile to deploy
-multiple clusters with unique requirements for security, networking, resource allocation, and so on.
+Use cluster profile variables to create placeholders for parameters in profile-layer configurations, which you can
+populate for individual clusters during deployment. With cluster profile variables, you can use a single cluster profile
+to deploy multiple clusters with unique requirements for security, networking, resource allocation, and more. You can
+also set specific constraints on the expected values, such as format, optionality, and masking to ensure scalable,
+error-free cluster deployments.
 
-When defining a cluster profile variable, you can set specific constraints on the expected values, such as format,
-optionality, masking, and so on, to ensure scalable, error-free cluster deployments.
+Cluster profile variables can be used with any Palette-connected cluster, including public cloud, data center, bare
```

Comment:
```suggestion
Cluster profile variables can be used with any Palette cluster, including public cloud, data center, bare
```
We can just say Palette cluster since we can use these for Edge and non-Edge